src/overlays/actors/ovl_En_Dekunuts/z_en_dekunuts.h

```diff
@@ -5,13 +5,34 @@
 
 struct EnDekunuts;
 
-typedef void (*EnDekunutsActionFunc)(struct EnDekunuts* this, GlobalContext* globalCtx);
+typedef void (*EnDekunutsActionFunc)(struct EnDekunuts*, GlobalContext*);
+
+#define ENDEKUNUTS_GET_FF00(thisx) (((thisx)->params >> 8) & 0xFF);
+
+enum {
+    /* 0x0 */ ENDEKUNUTS_GET_FF00_0,
+    /* 0x1 */ ENDEKUNUTS_GET_FF00_1,
+    /* 0x2 */ ENDEKUNUTS_GET_FF00_2,
+    /* 0xFF*/ ENDEKUNUTS_GET_FF00_FF = 0xFF,
```

Comment:
```suggestion
    /* 0x0 */ ENDEKUNUTS_GET_PARAMS_FF00_0,
    /* 0x1 */ ENDEKUNUTS_GET_PARAMS_FF00_1,
    /* 0x2 */ ENDEKUNUTS_GET_PARAMS_FF00_2,
    /* 0xFF*/ ENDEKUNUTS_GET_PARAMS_FF00_FF = 0xFF,
```
I think I'd still like an indication that these macros are related to params. Otherwise, it's very general and not clear

Reply:
My only input is stay consistent on the unknown params naming scheme.